c2corg_ui/static/js/search/outingfilters.js

```diff
@@ -64,7 +63,7 @@ app.OutingFiltersController.prototype.getFilterFromPermalink = function(key) {
     }
     var dates = val.split(',');
     dates.forEach(function(date) {
-      this.dates.push(app.utils.formatDate(date));
+      this.dates.push(window.moment(date)._d);
```

Comment:
`_d` is not exposed as API property/method from the authors of moment.js. So, they can simply rename/remove/change this property in a next release.